src/app/landing-page/landing-section/landing-section.component.html

```diff
@@ -0,0 +1,15 @@
+<section class="landing-section">
+    <div class="landing-section-text">
+        <h2>Choose a new phone easily!</h2>
+
+        <div class="landing-section-slogan">
+            <p>Simply answer a few questions and we will choose what is right for you!</p>
+            <p>Start using Mobite now!</p>
+            <button class="button button-large">Open app</button>
+        </div>
+
+        <div class="landing-section-image">
+            <img src="/assets/landing-page-phone.svg" />
+        </div>
+    </div>
+</section>
```

Comment:
File should have newline symbol at the end

Reply:
Done 0fce189f130739f9a0cdc92179ebeb82b357dd36